examples/hello_LoRa/hello_LoRa.ino

```diff
@@ -8,6 +8,7 @@
 // Copyright 2015, 2016 Ideetron B.V.
 //
 // Modified by Brent Rubell for Adafruit Industries, 2018
+// Added FramePort by clavisound for Adafruit Industries, Aug 2019
```

Comment:
Your contribution will be reflected in release notes, please don't add to the pre-existing license.